cypress/e2e/apitest/customers_request.cy.js

```diff
@@ -0,0 +1,100 @@
+        const sizes = ['Small','Medium','Enterprise','Large Enterprise','Very Large Enterprise']
+        it.each(sizes)('Filter to customer with size', (sizes) => {
+            cy.request(`${Cypress.env('baseUrl')}?page=1&size=${sizes}&industry=All`).then((response) => { 
+                expect(response.status).to.eq(200)
+                expect(response.body.customers[0].size).to.eq(sizes)
+            })
+        })
+    })
+    
+    context('Bad Requests', () => {
+        it('Check message for invalid page', () => {
+            cy.request({url: `${Cypress.env('baseUrl')}?page=0`, failOnStatusCode: false}).then((response) => { 
+                expect(response.status).to.eq(400)
+                expect(response.body.error).to.contain('Invalid page or limit. Both must be positive numbers.')
+            }) 
+        })
+        it('Check message for invalid limit', () => {
+            cy.request({url: `${Cypress.env('baseUrl')}?page=1&limit=0`, failOnStatusCode: false}).then((response) => { 
+                expect(response.status).to.eq(400)
+                expect(response.body.error).to.contain('Invalid page or limit. Both must be positive numbers.')
+            }) 
+        })
+        it('Check message for invalid page with insert string', () => {
+            cy.request({url: `${Cypress.env('baseUrl')}?page=teste&limit=0`, failOnStatusCode: false}).then((response) => { 
+                expect(response.status).to.eq(400)
+                expect(response.body.error).to.contain('Invalid page or limit. Both must be positive numbers.')
+            }) 
+        })
+        it('Check message for invalid limit with insert boolean', () => {
+            cy.request({url: `${Cypress.env('baseUrl')}?page=teste&limit=true`, failOnStatusCode: false}).then((response) => { 
+                expect(response.status).to.eq(400)
+                expect(response.body.error).to.contain('Invalid page or limit. Both must be positive numbers.')
+            })
+        })
+        it('Check message for invalid size', () => {
+            const message = 'Unsupported size value. Supported values are All, Small, Medium, Enterprise, Large Enterprise, and Very Large Enterprise.'
+            cy.request({ url: `${Cypress.env('baseUrl')}?page=1&limit=10&size=0`, failOnStatusCode: false}).then((response) => { 
+                expect(response.status).to.eq(400)
+                expect(response.body.error).to.contain(message)
+            })
+        })
+        it('Check message for invalid industry', () => {
+            const message = 'Unsupported industry value. Supported values are All, Logistics, Retail, Technology, HR, and Finance.'
+            cy.request({ url: `${Cypress.env('baseUrl')}?page=1&limit=10&size=All&industry=0`, failOnStatusCode: false}).then((response) => { 
+                expect(response.status).to.eq(400)
+                expect(response.body.error).to.contain(message)
+            })
+        })
+    })
+    
+})
```

Comment:
```suggestion
})

```

É recomendado deixar uma linha em branco ao final de cada arquivo para que ferramentas como o GitHub não exibam o ícone de `No newline at end of file`.

Isso é uma convenção e boa prática a ser seguida.